docs/reference/installation.rst

```diff
@@ -36,67 +37,34 @@ Use these commands:
     composer update
 
 Next, be sure to enable the bundles in your ``bundles.php`` file if they
-are not already enabled:
-
-.. code-block:: php
-
-    <?php
+are not already enabled::
 
     // config/bundles.php
 
     return [
-        //...
+        // ...
         FOS\CommentBundle\FOSCommentBundle::class => ['all' => true],
         Sonata\CoreBundle\SonataCoreBundle::class => ['all' => true],
         Sonata\BlockBundle\SonataBlockBundle::class => ['all' => true],
         Sonata\CommentBundle\SonataCommentBundle::class => ['all' => true],
     ];
 
-.. note::
-    If you are not using Symfony Flex, you should enable bundles in your
-    ``AppKernel.php``.
-
-.. code-block:: php
-
-    <?php
-
-    // app/AppKernel.php
-
-    public function registerbundles()
-    {
-        return array(
-            new FOS\CommentBundle\FOSCommentBundle(),
-            new Sonata\CoreBundle\SonataCoreBundle(),
-            new Sonata\BlockBundle\SonataBlockBundle(),
-            new Sonata\CommentBundle\SonataCommentBundle(),
-            // ...
-        );
-    }
-
 Configuration
 -------------
 
-.. note::
-    If you are not using Symfony Flex, all configuration in this section should
-    be added to ``app/config/config.yml``.
-
 SonataCommentBundle Configuration
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. code-block:: yaml
 
-    # config/packages/sonata.yaml
+    # config/packages/sonata_comment.yaml
 
     sonata_comment:
         manager_type: orm # can be 'orm' or 'mongodb'
         class:
             comment: App\Application\Sonata\CommentBundle\Entity\Comment # This is an optional value
```

Comment:
Please adjust this

Reply:
ping @OskarStark

Reply:
Ok, to what? `App\Entity\Comment`?

I mean there is no recipe for this bundle: https://github.com/symfony/recipes-contrib/tree/master/sonata-project
So we need to use EasyExtends command, right? And then the result is sth like this.

Please give me some feedback, I want to finish this PR!

Reply:
> Ok, to what? `App\Entity\Comment`?

Yes, because this docs suggest putting the class into the `App\Entity\` namespace